CHANGELOG.md

```diff
@@ -28,6 +28,80 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## v7.0.0-beta3 - 2018-11-26
+
+### [shopsys/coding-standards]
+
+#### Added
+
+- [#554] Add fixers to autocomplete missing return and param annotations, Thanks to [@TomasVotruba]
+
+### [shopsys/framework]
+
+#### Added
+
+- [#564] added a new translation extractor for properties of Constraints
+- [#546] added new Symfony command "shopsys:environment:change"
+- [#544] eased adding JS to admin
+
+#### Changed
+
+- [#576] `OrderFormType` in administration is now rendered by default
+
+#### Removed
+
+- [#595] removal of automatic price calculation
+
+#### Fixed
```

Comment:
fixed is somehow after removed and changed although `ch` is not the same as `c` `h` in english language